share/completions/cryptsetup.fish

```diff
@@ -1,92 +1,93 @@
-complete -c cryptsetup -l reduce-device-size -d "Reduce data device size (move data offset). DANGEROUS!"
-complete -c cryptsetup -l refresh -d "Refresh (reactivate) device with new parameters"
-complete -c cryptsetup -l resilience -d "Reencryption hotzone resilience type (checksum,journal,none)"
-complete -c cryptsetup -l resilience-hash -d "Reencryption hotzone checksums hash"
-complete -c cryptsetup -l resume-only -d "Resume initialized LUKS2 reencryption only"
-complete -c cryptsetup -l sector-size -d "Encryption sector size (default: 512 bytes)"
-complete -c cryptsetup -l serialize-memory-hard-pbkdf -d "Use global lock to serialize memory hard PBKDF (OOM workaround)"
-complete -c cryptsetup -l shared -d "Share device with another non-overlapping crypt segment"
-complete -c cryptsetup -l size -s b -d "The size of the device"
-complete -c cryptsetup -l skip -s p -d "How many sectors of the encrypted data to skip at the beginning"
-complete -c cryptsetup -l subsystem -d "Set subsystem label for the LUKS2 device"
-complete -c cryptsetup -l tcrypt-backup -d "Use backup (secondary) TCRYPT header"
-complete -c cryptsetup -l tcrypt-hidden -d "Use hidden header (hidden TCRYPT device)"
-complete -c cryptsetup -l tcrypt-system -d "Device is system TCRYPT drive (with bootloader)"
-complete -c cryptsetup -l test-passphrase -d "Do not activate device, just check passphrase"
-complete -c cryptsetup -l timeout -s t -d "Timeout for interactive passphrase prompt (in seconds)"
-complete -c cryptsetup -l token-id -d "Token number (default: any)"
-complete -c cryptsetup -l token-only -d "Do not ask for passphrase if activation by token fails"
-complete -c cryptsetup -l tries -s T -d "How often the input of the passphrase can be retried"
-complete -c cryptsetup -l type -s M -d "Type of device metadata: luks, luks1, luks2, plain, loopaes, tcrypt"
-complete -c cryptsetup -l unbound -d "Create unbound (no assigned data segment) LUKS2 keyslot"
-complete -c cryptsetup -l usage -d "Display brief usage"
-complete -c cryptsetup -l use-random -d "Use /dev/random for generating volume key"
-complete -c cryptsetup -l use-urandom -d "Use /dev/urandom for generating volume key"
-complete -c cryptsetup -l uuid -d "UUID for device to use"
-complete -c cryptsetup -l veracrypt -d "Scan also for VeraCrypt compatible device"
-complete -c cryptsetup -l veracrypt-pim -d "Personal Iteration Multiplier for VeraCrypt compatible device"
-complete -c cryptsetup -l veracrypt-query-pim -d "Query Personal Iteration Multiplier for VeraCrypt compatible device"
-complete -c cryptsetup -l verbose -s v -d "Shows more detailed error messages"
-complete -c cryptsetup -l verify-passphrase -s y -d "Verifies the passphrase by asking for it twice"
-complete -c cryptsetup -l version -s V -d "Print package version"
+complete -c cryptsetup -l active-name                        -d "Override device autodetection of dm device to be reencrypted"
```

Comment:
Please stop with the alignment - it's not worth it, and will be clobbered by `fish_indent`, our autoformatter.

See build_tools/style.fish.

Reply:
Sorry, I align it only on my side for better overview and forgot about `fish_indent` before pushing.